src/probleminstances/SolVSSGenerationTimetable.cpp

```diff
@@ -631,11 +620,11 @@ cda_rail::solver::mip_based::VSSGenTimetableSolver::extract_solution(
                        .at(breakable_edge_indices.at(e_index), vss)
                        .get(GRB_DoubleAttr_X),
                    ROUNDING_PRECISION);
-      if (debug) {
+      IF_PLOG(plog::debug) {
```

Comment:
A benefit I didn't mention in the previous review: The `IF_PLOG` is a macro, so it will be resolved during preprocessing. So, if no debug logging is done, this code won't even be in the compiled binary where previously the if-statement would have been there.